### ISSUE TITLE
Missing git tag for releases

### ISSUE THREAD

**Issue by lfrancke** [tapeinosyne/hyphenation#28]
Hi,

it seems as if tags are missing for later releases (last one is 0.6.0).
Would you mind tagging the newer releases?

Thank you!

**Comment by tapeinosyne** [comment]
Hi @lfrancke, thanks for letting me know.

Would tagging the latest version (v0.8.0) suffice for now, or do you need all versions to be tagged for a specific purpose?

**Comment by lfrancke** [comment]
Thanks for the quick response.

We ran a fossa.com check on our dependencies (it's all open source but I can't find a way to link to a FOSSA report with all the details) and it complains about hyphenation 0.7.1:
![image](https://user-images.githubusercontent.com/122850/104615547-a2fe1180-5689-11eb-8b9f-0689b93f2565.png)
This is one example: ![image](https://user-images.githubusercontent.com/122850/104615646-b8733b80-5689-11eb-8b68-57cd311aaae0.png)

So we wanted to check out the code at that revision and couldn't find it.
Which means we were especially interested in 0.7.1 but found it now. I opened this issue for the "general" problem as others might stumble across it as well.

It's not urgent at all though.

**Comment by tapeinosyne** [comment]
I went ahead and tagged all versions starting at v0.6, which should be enough.

On another note, if (L)GPL licensing is of concern to you, you might want to check out the brief discussion on #17. The short of it is that (L)GPL-licensed files are distributed as part of the `hyphenation` package, but never embedded in compiled build artifacts. If that is still problematic, do let me know.

(On yet another note, I am guessing `hyphenation` is not a direct dependency for you, but if it is, I'd be happy to open a PR to update to 0.8 — the API changes really are minimal.)

**Comment by lfrancke** [comment]
Thank you very much for tagging. I'll close this issue.

And also thanks for the link to the earlier issue. That's very helpful.

You're right. It's not a direct dependency for us:
![image](https://user-images.githubusercontent.com/122850/104698971-5f9bb580-5712-11eb-924a-9da64f5b054c.png)

I believe all the intermediate dependencies are libraries with compatible Cargo.toml version specifiers so it's up to us to update our binary dependencies. When we tried that we actually found out that this is a false-positive for us as `textwrap` includes `hyphenation` only as a non-default feature so we don't even have a dependency on `hyphenation`. FOSSA doesn't parse features correctly yet.

Thank you for the offer to help even further though and thanks for the quick turnaround.
